CONTRIBUTING.md

```diff
@@ -75,3 +75,77 @@
 +   3,
   ]
 ```
+
+## CSS Naming Conventions
+
+This is essentially our own flavour of BEM.
+
+* Use `.ComponentName` for components
+
+```html
+<style>
+  .Button {
+    color: var(--Button-fg);
+    background: var(--Button-bg);
+  }
+</style>
+
+<a class='Button'>Click me!</a>
+```
+
+* Use `&.\--modifier` for modifiers
+
+```html
+<style>
+  .Button {
+    color: var(--Button-fg);
+    background: var(--Button-bg);
+
+    &.\--invert {
+      color: var(--Button-bg);
+      background: var(--Button-fg);
+    }
+  }
+</style>
+
+<a class='Button --invert'>I'm more important</a>
+```
+
+* Use `&.is-state` for states (ie. temporary)
+
+```html
+<style>
+  .Image {
+    &.is-loading {
+      /* ... */
+    }
+  }
+</style>
+
+<div class='Image is-loading'></div>
+```
+
+* Use `&-childName` for children
```

Comment:
It seems that child component names begin with a lowercase, while parent components begin with an uppercase. Is this intentional? More importantly, is it a good idea?

Reply:
Yeah, it's intentional. I think it makes it more obvious what the parent and child are.